### PR TITLE
docs: Breaking changes docs

### PR DESCRIPTION
**Pull request by pictos**
Docs alerting users about the Breaking Change

### CODE REVIEW

File: doc/themes-breaking-changes.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+uid: Uno.Themes.BreakingChanges
+---
+
+# Overview
+
+The version 5.0.0 of Uno.Themes, introduced breaking changes on Converters, and Control Extensions. If you use one of those items on your project you will need to do some changes in order to keep using it.
```
Comment (Youssef1313) [inline doc/themes-breaking-changes.md:7]:
```suggestion
The version 5.0.0 of Uno.Themes introduced breaking changes on Converters and Control Extensions. If you use one of those items in your project, you will need to do some changes in order to keep using it.
```

File: doc/themes-breaking-changes.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## Converter
+
+All Converters were moved to Uno.Themes library, and the new `namespace` is `Uno.Themes`.
+
+Before:
+
+```xaml
```
Comment (Youssef1313) [inline doc/themes-breaking-changes.md:15]:
~~~suggestion
```xml
~~~

File: doc/themes-breaking-changes.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    <Page.Resources>
+        <um:FromNullToValueConverter x:Key="NotNullVisibilityConverter">
+    </Page.Resources>
+</Page>
+```
+
+After:
+
+```xaml
```
Comment (Youssef1313) [inline doc/themes-breaking-changes.md:26]:
~~~suggestion
```xml
~~~

File: doc/themes-breaking-changes.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+</Page>
+```
+
+## Control Extensions
+
+All Controls Extensions were moved to `Uno.Themes` library, and the new `namespace` is `Uno.Themes`.
+
+Before:
+
+```xaml
```
Comment (Youssef1313) [inline doc/themes-breaking-changes.md:41]:
~~~suggestion
```xml
~~~

File: doc/themes-breaking-changes.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+                <FontIcon Glyph="&#xE946;" />
+            </um:ControlExtensions.Icon>
+        </Button>
+    </StackPanel>
+</Page>
+```
+
+After:
+
+```xaml
```
Comment (Youssef1313) [inline doc/themes-breaking-changes.md:57]:
~~~suggestion
```xml
~~~

File: doc/themes-breaking-changes.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+                <FontIcon Glyph="&#xE946;" />
+            </ut:ControlExtensions.Icon>
+        </Button>
+    </StackPanel>
+</Page>
+```
+
+## Conclusion
+
+With that you should be able to update and get it working again for you app. If you've any other issue during the migration don't hesitate to open an issue on our [repo](https://github.com/unoplatform/Uno.Themes).
```
Comment (Youssef1313) [inline doc/themes-breaking-changes.md:73]:
```suggestion
With that, you should be able to update and get it working again for your application. If you've any other issues during the migration, don't hesitate to open an issue in [Uno.Themes GitHub repository](https://github.com/unoplatform/Uno.Themes).
```

File: doc/toc.yml [hunk 7]
```diff
@@ -2,6 +2,9 @@
 - name: Overview
   href: themes-overview.md
   items:
+- name: Breaking Changes
+  href: breaking-changes.md
```
Comment (Youssef1313) [inline doc/toc.yml:6]: File name not matching

File: doc/themes-breaking-changes.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
+---
+uid: Uno.Themes.BreakingChanges
```
Comment (kazo0) [inline doc/themes-breaking-changes.md:2]: This should be consolidated with the https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-migration.md page. So, a single Themes Migration page that will contain steps for Material/Cupertino/"design agnostic" migrations per Uno.Themes version

**Review by kazo0 — COMMENTED** [review]
Shouldn't this PR only contain docs changes? I see a lot of code changes as well, I thought those would only be applied in https://github.com/unoplatform/Uno.Themes/pull/1327

File: doc/themes-breaking-changes.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+---
+
+# Overview
+
+The version 5.0.0 of Uno.Themes, introduced breaking changes on Converters, and Control Extensions. If you use one of those items on your project you will need to do some changes in order to keep using it.
+
+## Converter
+
+All Converters were moved to Uno.Themes library, and the new `namespace` is `Uno.Themes`.
```
Comment (kazo0) [inline doc/themes-breaking-changes.md:11]: We should have a table of all converters with the old namespace and the new namespace like we do for other breaking change sections [here](https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-migration.md#resources)

File: doc/themes-breaking-changes.md
Comment (kazo0) [inline doc/themes-breaking-changes.md:1]: Merge this with https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-migration.md and break it up per major version with further breakdowns of Material/Cupertino (if any)/and general design-agnostic Themes lib changes

**Comment by pictos** [comment]
@kazo0 this PR was before the massive rebase, I'll address the comments here and cherry-pick with the main PR

**Comment by pictos** [comment]
Closing this one in favor of #1327

**Pull request closed without merging**